app/uk/gov/hmrc/basisperiodreformapi/connectors/BasisPeriodReformConnector.scala

```diff
@@ -40,4 +40,12 @@ class BasisPeriodReformConnector @Inject() (http: HttpClient, appConfig: AppConf
       queryParams = Seq(utr.map(("utr", _)), partnershipNumber.map(("partnership_ref_number", _))).collect { case Some(param) => param }
     ).map(enforcePartnership)
   }
+
+  def getSoleTrader(utr: Option[String])(implicit hc: HeaderCarrier): Future[TypedWrappedResponse[SoleTraderResponse]] = {
```

Comment:
Could this be called `getSoleTraderDetails` to match `getPartnershipDetails` above?